andela/ah-robotics
# #162949065 Integrate Travis to Coveralls Test Coverage

#### What does this PR do?
Integrate test coverage with Travis build result. 

#### Description of Task to be completed?
Install coveralls
Update the .travis.yml file with the  necessary configurations.

#### How should this be manually tested?
1. Open the project repository on GitHub here : [ah-robotics](https://github.com/andela/ah-robotics)
3. Switch to branch `ch-travis-integration-162949065`
4. Scroll down to the _**README**_ section
5. Click on the **_coverage_** badge to see the test coverage info on the coveralls.io website.
#### Any background context you want to provide?
None

#### What are the relevant pivotal tracker stories?
[#162949065](https://www.pivotaltracker.com/story/show/162949065)

#### Screenshot
**Coverage report on coveralls**
<img width="1440" alt="screen shot 2019-01-10 at 12 10 14 am" src="https://user-images.githubusercontent.com/10683903/50928594-2453b580-146c-11e9-97cc-d8056ef1d428.png">



#### Questions:
Feel free to give feedback and raise any questions.
